sound/soc/intel/boards/bytcht_es8316.c

```diff
@@ -38,6 +38,7 @@ struct byt_cht_es8316_private {
 	struct snd_soc_jack jack;
 	struct gpio_desc *speaker_en_gpio;
 	bool speaker_en;
+	struct device *codec_dev;
```

Comment:
Better to put before bool, on x86 it may save 4 bytes.

Reply:
ack